source/docs/solar/ncdb/index.html.md.erb

```diff
@@ -0,0 +1,17 @@
+---
+title: Climate Viewer Data Downloads
+summary: Freely available downloads of The National Solar Radiation Database.
```

Comment:
We need to ask the team what copy they want here as well as in the description below